lib/NodeStuffPlugin.js

```diff
@@ -17,6 +17,8 @@ class NodeStuffPlugin {
 
 	apply(compiler) {
 		const options = this.options;
+		if (options === false) return;
```

Comment:
It should never be applied. Wrap the plugin in an `if` in `WebpackOptionsApply.js`. Require the plugin conditionally.